### DISCUSSION TITLE
DataView: Can not style Paginator

### DISCUSSION THREAD

**Discussion by CESolomon** [primefaces/community#2869] [PrimeVue]
I am using the DataView in my Laravel11+Inertia+Vue3 project. I would like to style the Paginator elements (such as changing background color, font/icon control/link colors and sizes. However, nothing seems to work to affect it. I have tried using ":pt" values but nothing changes it. I also tried changing the CSS classes as scope styles (ex. p-paginator) and that had no effect either. Any help?

**Comment by j0rgedev** [comment]
Hey @CESolomon it would be great if you share a repro or the way your trying to customize the Paginator component

**Reply by CESolomon** [reply]
:pt="{
                    paginator: {
                         root: 'bg-red-300',
                         current: 'border border-gray-200 rounded-full',
                    },
                }"

**Reply by j0rgedev** [reply]
By the way, are u using primevue v3 or v4?

**Reply by CESolomon** [reply]
4.0.4

**Reply by j0rgedev** [reply]
Have you already tried customizing it with its theming tokens? https://primevue.org/paginator/#theming.tokens. Haven't tried it with paginator specifically, but I have with other components, so it should work
